javascript/src/components/streamrules/HumanReadableStreamRule.jsx

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var React = require('react/addons');
+
+var HumanReadableStreamRule = React.createClass({
+    _getTypeForInteger(type, streamRuleTypes) {
+        if (streamRuleTypes) {
+            return streamRuleTypes.filter((streamRuleType) => {
+                return String(streamRuleType.id) === String(type);
+            })[0];
+        } else {
+            return undefined;
+        }
+    },
+    _formatRuleValue(streamRule) {
+        if (streamRule.type !== 5) {
```

Comment:
It is the type value of a "field presence" rule. In this special case the value part is not displayed. Maybe I should extract it to a constant.